administrator-guides/omnichannel-integrations/livechat-widget/livechat-enterprise/canned-responses/README.md

```diff
@@ -0,0 +1,36 @@
+# Canned Responses
+
+With canned responses, Enterprise users can send response templates tp LiveChat and other rooms.
+
+A canned response includes two attributes:
+
+* `shortcut` - a name beginning with an exclamation mark (`!`), used to call the template.
+* `text` - the content users enter into the Text field.
+
+## Enable Canned Responses
+
+To enable canned responses go to **Administration > Canned Responses**
+
+[image](administrator-guides\omnichannel-integrations\livechat-widget\livechat-enterprise\canned-responses\enable-canned-responses.png)
+
+By default, the livechat-agent, livechat-monitor, and admin roles receive permissions to save, delete and view canned responses:
+
+[image](administrator-guides\omnichannel-integrations\livechat-widget\livechat-enterprise\canned-responses\canned-responses-permissions.png)
+
+## Create a New Canned Response
```

Comment:
It's possible to create `Canned Responses` per departments, I think we should mention and describe it here.